host/frontend/host-orchestrator/main.go

```diff
@@ -138,8 +153,14 @@ func main() {
 	orchestrator.SetupInstanceManagement(r, im, om)
 	// The host orchestrator currently has no use for this, since clients won't connect
 	// to it directly, however they probably will once the multi-device feature matures.
-	fs := http.FileServer(http.Dir(DefaultStaticFilesDir))
-	r.PathPrefix("/").Handler(fs)
+	if use_ng_serve {
+		angularUrl, _ := url.Parse("http://localhost:4200")
```

Comment:
> Looks good in general, but this may be unnecessary. To speed up developer work you have a couple of options:
> 
> * Use bind mount:
>   
>   * stop the service running in your machine
>   * back up the installed webui dir
>   * bind-mount the developer output directory to where the service expects to find the web UI
>   * restart the service
> * Run locally:
>   
>   * stop the service
>   * give yourself write access in `/run/cuttlefish`
>   * build the operator/orchestrator binary (use the same script used by debian build)
>   * run the generated binary
> 
> Both approaches work because the file server doesn't cache files, so any modifications/additions/deletions should be visible in the browser immediately.

Thanks for the suggestions. Yeah, both can speed up dev cycle. But one of the benefits `ng serve` over those options is that it can watch src, and auto-reload without explicit build command, and also it supports hot module replacement.

Reply:
SGTM, replaced it with url.